CMakeLists.txt

```diff
@@ -14,6 +14,9 @@
 #                                                                         #
 ###########################################################################
 
+# Mandatory call to project
+project(opm-material CXX)
```

Comment:
*blush* how could i have missed this for all these years. this is definitely correct.